src/components/YourCompany/YourCompany.vue

```diff
@@ -155,41 +271,80 @@ export default class YourCompany extends Mixins(DateMixin, EntityFilterMixin, Le
   private correctNameChoices: Array<string> = []
   private isEditingNames = false
 
+  /**  Initialize the name choices for alterations */
+  mounted () { this.onApprovedName() }
+
   /** The company name (from NR, or incorporation number). */
   private get companyName (): string {
     if (this.getApprovedName) return this.getApprovedName
 
     return `${this.getBusinessNumber || '[Incorporation Number]'} B.C. Ltd.`
   }
 
+  private get expiryDate (): string {
+    return new Date(this.getNameRequest.expiry).toDateString()
```

Comment:
We always get in trouble here...
- Does this correctly convert UTC -> Pacific?
- Is this a consistent display date format within this project (and other Entity projects)?

Reply:
Indeed we do, we seem to get a lot of different formats to deal with. 

This correctly converts the time given by the API to local time, as an example the expiry is set at Wed Mar 10 2021 07:59 GMT... which will convert into the day prior 1 minute before the end of the day. 

That being said, i think that is correct, as it WILL expire at the very end of the day that is displayed. 

To the last bullet, i'm not entirely sure. It is consistent with the design but i think we have some instances in the Filings/Create UI that are still in an older format.

Reply:
IIRC, the NR expires at 00:00:00 on its expiry date,

Reply:
... which is counterintuitive.

We handle that by displays "days left" and "expires tomorrow" in the Filings UI. Might need to do that here if it's not clear. Ask UX for guidance if needed.

Reply:
Hmm, yea. I will bring it up during the UX review. Potentially a verbiage change there.

Reply:
Did you want me to action anything here? 
As you mentioned, expiring at 00:00:00 on the date we are given would mean displaying the Date at GMT not PST, since after conversion it becomes the day prior due to the Offset in the given date.

Reply:
As it is, the expiry date shown will be the last day it's useable, if we don't convert to local then it would become NOT useable on the date displayed. 
Sorry if there was a conversation around this preference earlier 😁

Reply:
Now I'm thoroughly confused. For the record.

Reply:
As long as:
- the NR is expired as of 00:00:00 on its expiry date, and
- the NR expiry date is correctly displayed to the user
Then I think we're good.